.travis.yml

```diff
@@ -53,9 +39,13 @@ jobs:
         - "bin/installDeps.sh"
         - "export GIT_HASH=$(git rev-parse --verify --short HEAD)"
       script:
-        - "tests/frontend/travis/runner.sh"
+        - "travis_wait 15 tests/frontend/travis/runner.sh"
     - name: "Run the Backend tests without Plugins"
       install:
+        - "sudo add-apt-repository -y ppa:libreoffice/ppa"
+        - "sudo apt-get update"
+        - "sudo apt-get -y install libreoffice"
+        - "sudo apt-get -y install libreoffice-pdfimport"
```

Comment:
Maybe factor this out to avoid duplication?  Something like this:

```yaml
_install_libreoffice: &install_libreoffice >-
  sudo add-apt-repository -y ppa:libreoffice/ppa &&
  sudo apt-get update &&
  sudo apt-get -y install libreoffice libreoffice-pdfimport

jobs:
  include:
    - name: "Run the Backend tests without Plugins"
      install:
        - *install_libreoffice
        - "bin/installDeps.sh"
        # ...
```

Reply:
I decided to merge prior to making this change, at least we can get some runs in today and then make this change when things are stable